ee/clickhouse/views/test/test_clickhouse_insights.py

```diff
@@ -172,6 +172,39 @@ def test_breakdown_with_filter(self):
         person_response = self.client.get("/" + response["result"][0]["persons"]["url"]).json()
         self.assertEqual(len(person_response["results"][0]["people"]), 1)
 
+    def test_insight_trends_compare(self):
+        with freeze_time("2012-01-05T03:21:34.000Z"):
+            _create_person(distinct_ids=["1"], team=self.team)
+            _create_person(distinct_ids=["2"], team=self.team)
+            _create_event(team=self.team, event="$pageview", distinct_id="1", properties={"key": "val"})
+            _create_event(team=self.team, event="$pageview", distinct_id="2", properties={"key": "notval"})
+
+        with freeze_time("2012-01-14T03:21:34.000Z"):
+            _create_event(team=self.team, event="$pageview", distinct_id="1", properties={"key": "val"})
+            _create_event(team=self.team, event="$pageview", distinct_id="2", properties={"key": "notval"})
+
+        with freeze_time("2012-01-15T04:01:34.000Z"):
+            data = deep_dump_object(
+                {
+                    "date_from": "-7d",
+                    "compare": True,
+                    "events": [{"id": "$pageview", "name": "$pageview", "type": "events", "order": 0,}],
+                }
+            )
+            response = self.client.get(f"/api/projects/{self.team.id}/insights/trend/", data=data).json()
+
+        self.assertEqual(response["result"][0]["data"], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 0.0])
```

Comment:
non-blocking

When I look at these tests, I think the main thing I want to do is try and match up the events created above with the numbers in the graph. I think the most obvious way to do that is via the timestamp of the events, but perhaps there is another way to make this easy?

i.e. have something like 

```
assert data_by_date == {
  "2012-01-11": 0.0,
  "2012-01-12": 2.0,
  "2012-01-12": 0.0
}
```

Reply:
yeah this needs to be updated after finishing the util in the other PR

Reply:
cool 👍